css/css-transforms/META.yml

```diff
@@ -0,0 +1,83 @@
+links:
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=206514
+  results:
+  - test: perspective-origin-001.html
+    status: FAIL
+  - test: perspective-origin-002.html
+    status: FAIL
+  - test: perspective-origin-003.html
+    status: FAIL
+  - test: perspective-origin-004.html
+    status: FAIL
+  - test: perspective-origin-005.html
+    status: FAIL
+  - test: perspective-origin-006.html
+    status: FAIL
+  - test: perspective-origin-x.html
+    status: FAIL
+  - test: perspective-origin-xy.html
+    status: FAIL
+  - test: perspective-translateZ-0.html
+    status: FAIL
+  - test: perspective-translateZ-negative.html
+    status: FAIL
+  - test: perspective-translateZ-positive.html
+    status: FAIL
+  - test: text-perspective-001.html
+    status: FAIL
+  - test: transform-3d-rotateY-stair-above-001.xht
+    status: FAIL
+  - test: transform3d-matrix3d-005.html
+    status: FAIL
+  - test: transform3d-perspective-006.html
+    status: FAIL
+  - test: transform3d-preserve3d-011.html
+    status: FAIL
+  - test: transform3d-scale-004.html
+    status: FAIL
+  - test: transform3d-sorting-001.html
+    status: FAIL
+  - test: transofrmed-preserve-3d-1.html
```

Comment:
Ahaha, I am amused to find that this is not a typo. (As in, a test with that name really exists.)

Reply:
do you think is it worth to rename the test on WPT?

Reply:
Nah, I wouldn't bother. If anyone ever does, we will pick it up via the path-checking test in wpt-metadata.

Reply:
Opened PR https://github.com/web-platform-tests/wpt/pull/21394 to rename the tests

Reply:
> Opened PR [web-platform-tests/wpt#21394](https://github.com/web-platform-tests/wpt/pull/21394) to rename the tests

Ups, I did it before seeing your previous comment. sorry.
Let's continue this PR with the current name and if the other PR ends landing I will update here accordingly.